apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListFeatureExtractorSamples.kt

```diff
@@ -64,4 +64,28 @@ class ListFeatureExtractorSamples {
             }
         }
     }
+
+    @Test
+    fun last() {
```

Comment:
split this into two please, use `last` (this fun) for explaining how the `fun last` works and `lastFeature` to explain how the `val last` works. see get and getFeature above